platform-tests/src/platform/acceptance/compose_redis_service_test.go

```diff
@@ -0,0 +1,84 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/helpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Compose Redis backing service", func() {
+	const (
+		serviceName  = "compose-redis"
+		testPlanName = "tiny"
+	)
+
+	It("is registered in the marketplace", func() {
+		plans := cf.Cf("marketplace").Wait(testConfig.DefaultTimeoutDuration())
+		Expect(plans).To(Exit(0))
+		Expect(plans).To(Say(serviceName))
+	})
+
+	It("has the expected plans available", func() {
+		plans := cf.Cf("marketplace", "-s", serviceName).Wait(testConfig.DefaultTimeoutDuration())
+		Expect(plans).To(Exit(0))
+		Expect(plans.Out.Contents()).To(ContainSubstring("tiny"))
+	})
+
+	PContext("creating a database instance", func() {
```

Comment:
Do you need that `PContext`? I see you took it out from compose?

Reply:
In the Compose tests it was originally skipped because we had random 5xx errors from their API. It's just a little bit confusing in the diff as this file was just renamed.